pkg/operator/discover/discover.go

```diff
@@ -86,6 +87,12 @@ func (d *Discover) createDiscoverDaemonSet(ctx context.Context, namespace, disco
 		discoveryParameters = append(discoveryParameters, "--use-ceph-volume")
 	}
 
+	discoverDaemonResourcesRaw := os.Getenv(discoverDaemonResourcesEnv)
+	discoverDaemonResources, err := k8sutil.YamlToContainerResource(discoverDaemonResourcesRaw)
```

Comment:
To follow up on discussion in huddle, we could potentially add this to the configmap settings instead of env var, but probably actually better to just keep it in the env var settings for now unless there is a need to move all the discovery daemon settings to the configmap at the same time.

Reply:
Sure